css/styles.css

```diff
@@ -73,6 +73,16 @@ html, body {
   opacity: .7;
 }
 
+.mute-audio {
+  position: fixed;
+  font-size: 1.3rem;
+  margin: 8px;
+  color: white;
+  bottom: 10%;
```

Comment:
In this case we'd be better off using fixed pixels for positioning.
I suspect this might break for some devices.

Reply:
I tried with a couple of devices (>10) from Chrome Dev Tools Emulator. Things are looking better when given in %. I guess this depends on perspective.

Reply:
Please attach some screenshots for iPhone X, pixel, moto 4 in both landscape and portrait mode.